### PR TITLE
Add GTON Network mainnet and testnet info

### PR DESCRIPTION
**Pull request by drinkius**

### CODE REVIEW

File: _data/chains/eip155-1000.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://gton.capital",
+  "shortName": "gton",
+  "chainId": 1000,
+  "networkId": 1000,
+  "explorers": [
+    {
+        "name": "GTON Network Explorer",
+        "url": "https://explorer.gton.network/",
```
Comment (ligi) [inline _data/chains/eip155-1000.json:20]:
```suggestion
        "url": "https://explorer.gton.network",
```

File: _data/chains/eip155-50021.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://gton.capital",
+  "shortName": "gton",
+  "chainId": 50021,
+  "networkId": 50021,
+  "explorers": [
+    {
+        "name": "GTON Testnet Network Explorer",
+        "url": "https://explorer.testnet.gton.network/",
```
Comment (ligi) [inline _data/chains/eip155-50021.json:20]:
```suggestion
        "url": "https://explorer.testnet.gton.network",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `gton` more than once

**Comment by drinkius** [comment]
> short name must be unique - but found `gton` more than once

Sure, will fix that

**Comment by drinkius** [comment]
> short name must be unique - but found `gton` more than once

Fixed

**Pull request merged**